cdtb/storage.py

```diff
@@ -88,6 +88,12 @@ def __init__(self, v: Union[str, tuple]):
                 self.s = '.'.join(str(x) for x in self.t)
             assert len(self.t) == 2, "invalid patch version format"
 
+    def __int__(self):
```

Comment:
I'm not a fan of implementing this with `__int__`.  The returned number is not a "normal" integer that can be added to other numbers. It's some sort of special value. (We could actually use a `(x, y)` tuple btw.)

A dedicated, non-magic method would be better IMHO.

Reply:
Why not this?
```py
def version_as_int(s: str) -> int:
    if s == "main":
        return 9999
    x, y = s.split('.', 2)
    return 100 * int(x) + int(y)
```
Then use
```py
# In export code
game_version = version_as_int(self.patch.version.s)
# In arenadata.py
parsed_version = version_as_int(args.patch_version)
```
I made it simple but `version_as_int()` could be adjusted to accept more types and values (`latest`, `None`, a `PatchVersion`, etc.).